TextToSpeech/impl/TTSConfiguration.h

```diff
@@ -75,6 +77,8 @@ class TTSConfiguration {
     int8_t m_primVolDuck;
     bool m_preemptiveSpeaking;
     bool m_enabled;
+    bool m_AclCalled;
```

Comment:
why do we need this redundant flag when we can simply check if there are any valid elements in the ACL map?

Reply:
addressed on https://github.com/rdkcentral/rdkservices/pull/4359